src/videojs.ads.js

```diff
@@ -113,12 +114,14 @@ var
           updateEnded = function() {
             ended = true;
           };
-        player.currentTime(snapshot.currentTime);
```

Comment:
So `snapshot.currentTime` was not returning the actual video duration in some cases before a postroll? Any details on where/why this was occurring?